apps/scribe/README.md

```diff
@@ -8,7 +8,7 @@ Make sure you configure the webhook according to the steps in the [CMS readme](.
 
 Duplicate `.env.sample` > `.env` in the `scribe` folder and enter the required environment variables. Make sure you've created a Postgres databases that matches what's set in the `DATABASE_URL` key in your `api/.env` file.
 
-_NOTE_: The `scribe/.env` and the `api/.env` are almost identical, and will share most values.
+_NOTE_: The `scribe/.env` and the `api/.env` are almost identical and will share most values, the notable exception being the `PORT` value in the # API Configuration. These must run on separate ports.
```

Comment:
`# API Configuration` here should probably say `# Scribe Configuration` based on the change in the above file